soknad/src/main/java/no/nav/k9/søknad/ytelse/psb/v1/PleiepengerSyktBarnValidator.java

```diff
@@ -36,16 +34,21 @@
                 .map(this::toFeil)
                 .collect(Collectors.toList());
 
-        validerSøknadsperiode(psb.getSøknadsperiode(), feil);
-        var søknadsperiode = psb.getSøknadsperiode();
+        try {
+            if(psb.getSøknadsperiode() != null) {
```

Comment:
når er søknadsperiode null? da vil det ikke være mulig å korrelere med en sak?

Reply:
Kan ikke se for meg at en søknad uten søknadsperiode gir menig. Så er helt greit med en NullPointerException.